modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2378,7 +2389,8 @@ private void _addAccountEntryOrganizationRel(
 	}
 
 	private void _addAggregationObjectField(
-			String functionName, String objectFieldName,
+			String aggregationObjectFieldName, String functionName,
+			String objectFieldName, long objectDefinitionId,
```

Comment:
@pedro-oliveira446 rename objectFieldName to argumentObjectFieldName and rename aggregationObjectFieldName to objectFieldName,

Thanks!